### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by Cliftonz**

### CODE REVIEW

File: _includes/resources.html [hunk 1]
```diff
@@ -22,18 +22,18 @@ <h4>Reading</h4>
...
             </li>
             <br/>
             <li>
-                <a href="https://www.nationalccdc.org/">CCDC</a> -- The Colegent Cyber Defence Competion is a team based competition were the team is dropped into a live environment that is being actively hacked. 
+                <a href="https://www.nationalccdc.org/">CCDC</a> -- The Collegiate Cyber Defence Competion is a team based competition were the team is dropped into a live environment that is being actively hacked. 
             </li>
             <br/>
             <li>
-                <a href="https://www.nationalcyberleague.org/">NCL</a> -- The National Cyber league is the first steps into learning cybersecurity, were the participates are guided through all topics in cybersecurity.
+                <a href="https://www.nationalcyberleague.org/">NCL</a> -- The National Cyber League is the first steps into learning cybersecurity, were the participates are guided through all topics in cybersecurity.
```
Comment (TheOtherUnknown) [inline _includes/resources.html]: Typo
```suggestion
                <a href="https://www.nationalcyberleague.org/">NCL</a> -- The National Cyber League is the first steps into learning cybersecurity, were the participants are guided through all topics in cybersecurity.
```

File: _includes/resources.html [hunk 2]
```diff
@@ -22,18 +22,18 @@ <h4>Reading</h4>
...
         </ul>
-        <h4>Competions</h4>
+        <h4>Competitions</h4>
         <ul class="resourceitem">
             <li>
                 <a href="https://open.kattis.com/">Kattis (ICPC)</a> -- The International Collegiate Programming Contest is team based competion were a group of three people go and try to complete as many programing based problems as they can.
             </li>
             <br/>
             <li>
-                <a href="https://www.nationalccdc.org/">CCDC</a> -- The Colegent Cyber Defence Competion is a team based competition were the team is dropped into a live environment that is being actively hacked.
```
Comment (TheOtherUnknown) [inline _includes/resources.html]: You should probably use mdashes for those instead of double-hyphens.

File: _includes/footer.html [hunk 3]
```diff
@@ -1,7 +1,9 @@
+
 <footer>
     <div class="bml">
         <section class="container">
-            <p class="text-center"> Copyright &copy; 2019 IUS</p>
+            <p class="text-center"> Copyright &copy; 2020 IUS</p>
+            <h6 class="footer-text">The CSG will sponsor a student for class particpation if the student attendeds 2 lectures, participates in a competition, or becomes an officer.</h6>
```
Comment (TheOtherUnknown) [inline _includes/footer.html]: I think that should go somewhere other than the footer. Maybe another page?

File: _includes/getinvolved.html [hunk 4]
```diff
@@ -8,9 +8,11 @@ <h2>Get Involved</h2>
             <a class="btn btn-primary" href="https://list.ius.edu/sympa/subscribe/csg-announce-l">Join our mailing list</a>
             <a class="btn btn-primary" href="index.html#schedule">Attend an event</a>
             <a class="btn btn-primary" href="https://discord.gg/XGfzKxA">Join our Discord Server</a>
-            <a class="btn btn-primary" href="https://trello.com/invite/iuscomputerscigroup/9c12b7af83c5e84cef44fdb5d1f2d9e5">Join our Tello boads</a>
+            <a class="btn btn-primary" href="https://trello.com/invite/iuscomputerscigroup/9c12b7af83c5e84cef44fdb5d1f2d9e5">Join our Trello boards</a>
             <a class="btn btn-primary" href="index.html#resources">Read our recommended resources</a>    
         </p>
+        <br/>
+        <br/>
```
Comment (TheOtherUnknown) [inline _includes/getinvolved.html]: Doing this with CSS padding would probably be better.

File: _includes/footer.html (hunk 3 above)
Comment (Cliftonz) [reply _includes/footer.html]: moved to its own section

File: _includes/participation.html [hunk 5]
```diff
@@ -0,0 +1,6 @@
+
+<div>
+    <h4 style="text-align: center;">Participation Credit</h4>
+    <h6 class="footer-text">The CSG will sponsor a student for class particpation if the student attends 2 lectures, participates in a competition, or becomes an officer.</h6>
```
Comment (TheOtherUnknown) [inline _includes/participation.html]: This text is very small style wise. Why are you using h6?
I still think it should be it's own page, but if you can get someone else to approve it go ahead.

**Comment by TheOtherUnknown** [comment]
@Cliftonz Any updates on this?

**Pull request closed without merging**